walk-in-interview/src/test/java/com/google/job/data/JobsDatabaseTest.java

```diff
@@ -0,0 +1,102 @@
+package com.google.job.data;
+
+import com.google.api.core.ApiFuture;
+import com.google.cloud.firestore.*;
+import com.google.firestore.FireStoreUtils;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+
+public final class JobsDatabaseTest {
+
+    private static final String TEST_JOB_COLLECTION = "Jobs";
+    private static final int BATCH_SIZE = 10;
+
+    Firestore firestore;
+    JobsDatabase jobsDatabase;
+
+    @Before
+    public void setUp() throws IOException {
+        firestore = FireStoreUtils.getFireStore();
```

Comment:
With the current config setup, this will use development configuration right? Anyway we can switch it to test config?